dubbo-dependencies-bom/pom.xml

```diff
@@ -571,6 +574,19 @@
                 <artifactId>testcontainers</artifactId>
                 <version>${test_container_version}</version>
             </dependency>
+
+            <dependency>
+                <groupId>javax.annotation</groupId>
+                <artifactId>javax.annotation-api</artifactId>
+                <version>${javax_annotation_api_version}</version>
+            </dependency>
+
+            <dependency>
+                <groupId>junit</groupId>
+                <artifactId>junit</artifactId>
+                <version>${junit_version}</version>
+                <scope>test</scope>
+            </dependency>
```

Comment:
Junit 4 is not necessary. 
Junit 5 is included in the parent pom and it can be used directly inside the new module without any extra declaration.

Reply:
ok. i'll changed